colossus/src/main/scala/colossus/protocols/http/Header.scala

```diff
@@ -26,6 +29,30 @@ object HttpMethod {
 
   val methods: List[HttpMethod] = List(Get, Post, Put, Delete, Head, Options, Trace, Connect, Patch)
 
+  def apply(line: Array[Byte]): HttpMethod = {
```

Comment:
I wouldn't mind seeing some tests that demonstrate this rejecting bad HTTP request with bad verbs